flashcards/backend/decks/tests/test_model_tag.py

```diff
@@ -0,0 +1,17 @@
+from ..models import Tag, Flashcard, Deck
+
+
+def test_tag_gui_representation():
+    tag = Tag(name="testTag")
+
+    assert str(tag) == "testTag"
+
+
+def test_tag_in_db(db):
+    tag = Tag(
+        name="testTag2", description="testDescription", flashcard=Flashcard.objects.create(), deck=Deck.objects.create()
```

Comment:
I would Create fixture for Deck and flashcard

Reply:
Done.

Reply:
sorry don't see the change you are still crating objects with `Flascard.objects.create()` what I meant was to create a pytest fixture and use it here in test